Tribler/Core/Modules/restapi/display_endpoint.py

```diff
@@ -0,0 +1,89 @@
+import json
+
+from twisted.web import http, resource
+
+
+class DisplayEndpoint(resource.Resource):
+    """
+    This endpoint is responsible for handling requests for the trust display.
+    """
+
+    def __init__(self, session):
+        """
+        Initialize the DisplayEndpoint
+        :param session: the Session object where the aggregated data can be retrieved from
+        """
+        self.session = session
+        resource.Resource.__init__(self)
+
+    @staticmethod
+    def return_400(request, message="your request seems to be wrong"):
+        """
+        Return a HTTP Code 400 with the given message
+        :param request: The request which has to be changed
+        :param message: The error message which is used in the JSON string
+        :return: The error message formatted in JSON
+        """
+        request.setResponseCode(http.BAD_REQUEST)
+        return json.dumps({"error": message})
+
+    def render_GET(self, request):
+        """
+        Process the GET request which retrieves the information used by the GUI Trust Display window
+        
+        .. http:get:: /display?focus_node=(string: identifier)&neighbour_level=(int: neighbour_level
+        
+        A GET request to this endpoint returns the data from the multichain. This data is retrieved from the multichain
+        database and will be focused around the given focus node. The neighbour_level parameter specifies which nodes
+        are taken into consideration (e.g. a neighbour_level of 2 indicates that only the focus node, it's neighbours
+        and the neighbours of those neighbours are taken into consideration). Please note that the neighbour_level
+        parameter is optional; if the parameter is not set, a neighbour_level of 1 is assumed.
+        
+        The returned data will be in such format that the GUI component which visualizes this data can easily use it.
+        Although this data might not seem as formatted in a useful way to the human eye, this is done to accommodate as
+        little parsing effort at the GUI side.
+        
+            **Example request**:
+            
+            .. sourcecode:: none
+            
+                curl -X GET http://localhost:8085/display?focus_node=xyz
+            
+            **Example response**:
+            
+            .. sourcecode:: javascript
+            
+                {
+                    "focus_node": "xyz",
+                    "neighbour_level: 1
+                    "nodes": [{
+                        "name": "xyz",
```

Comment:
Change the 'name' field to something like 'public_key' or 'pk', since the nodes are defined by public key, not by name.
(Same for 'focus_node' on line 57)